packages/redux-devtools-inspector-monitor/src/tabs/StateTab.tsx

```diff
@@ -16,18 +18,39 @@ const StateTab: React.FunctionComponent<
   labelRenderer,
   dataTypeKey,
   isWideLayout,
-}) => (
-  <JSONTree
-    labelRenderer={labelRenderer}
-    theme={getJsonTreeTheme(base16Theme)}
-    data={nextState}
-    getItemString={(type, data) =>
-      getItemString(styling, type, data, dataTypeKey, isWideLayout)
-    }
-    invertTheme={invertTheme}
-    hideRoot
-  />
-);
+}) => {
+  const [sortObjectKeys, setSortObjectKeys] = useState(false);
+  const [disableCollection, setDisableCollection] = useState(false);
+
+  useEffect(() => {
+    if (!chrome || !chrome.storage) return;
+    const storage = isFF
+      ? chrome.storage.local
+      : chrome.storage.sync || chrome.storage.local;
+    storage.get(
+      ['sortStateTreeAlphabetically', 'disableStateTreeCollection'],
+      function (result) {
+        setSortObjectKeys(!!result.sortStateTreeAlphabetically);
+        setDisableCollection(!!result.disableStateTreeCollection);
+      }
+    );
+  }, []);
```

Comment:
Can we make it so these are props instead? Only the extension itself should be reading/writing settings from storage. It should then pass those settings down as props.

Reply:
Sure. Will adjust it in the following days.